monero-harness/src/image.rs

```diff
@@ -78,7 +78,7 @@ impl Image for MoneroWalletRpc {
     type EntryPoint = str;
 
     fn descriptor(&self) -> String {
-        "melotools/monero:v0.17.2.0".to_owned()
+        "rinocommunity/monero:v0.17.2.0".to_owned()
```

Comment:
This need to be changed to 17.3.0 because there is no tag for 17.2.0 Rinocommunity